### PR TITLE
changes for DALI v1.3.0

### PR DESCRIPTION
**Pull request by cdeepali**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

https://github.com/open-ce/DALI-feedstock/issues/23

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: envs/conda_build_config.yaml [hunk 1]
```diff
@@ -139,7 +139,7 @@ libtiff:
 libtool:
   - 2.4.6
 lmdb:
-  - 0.9.22.*
+  - 0.9.29.*
```
Comment (jayfurmanek) [inline envs/conda_build_config.yaml:142]: Maybe we just loosen this pin?

**Review by jayfurmanek — APPROVED** [review]

**Comment by jayfurmanek** [comment]
Do we need a pin for `libvorbis` as well?

File: envs/conda_build_config.yaml (hunk 1 above)
Comment (cdeepali) [reply envs/conda_build_config.yaml:144]: Yes, it is needed. I had it in my local copy where I was also using changes for the protobuf refresh. Added the change here now.

**Comment by cdeepali** [comment]
Yes, `libvorbis pin` is needed. I had it in my local copy where I was also using changes for the protobuf refresh. Added the change here now.

File: envs/conda_build_config.yaml (hunk 1 above)
Comment (cdeepali) [reply envs/conda_build_config.yaml:144]: yes we can do so. Updating it.

**Review by jayfurmanek — APPROVED** [review]

**Pull request merged**